packages/schema-dot-org-json-ld-components/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "schema-dot-org-json-ld-components",
+  "version": "0.1.0",
+  "description": "Components to generate schema.org json-ld markup.",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha --compilers js:babel-core/register"
+  },
+  "keywords": [
+    "schema.org",
+    "jsonld",
+    "react"
+  ],
+  "author": "John Bito (http://jwbito.ballardview.com/)",
+  "license": "UPL-1.0",
+  "devDependencies": {
+    "babel-core": "^6.25.0",
```

Comment:
Unless you're calling a CLI, you can omit these devDependencies and place them in the root package.json. Node will find them due to the recursive `node_modules` lookup algorithm.